repro.in

```diff
@@ -528,14 +531,20 @@ General Options:
  -d                           Run diffoscope if packages are not reproducible
  -f                           Use the local PKGBUILD for building
  -n                           Run makepkg with --nocheck
+ -V                           Print version information
  -o <path>                    Set the output directory (default: ./build)
 __END__
 }
 
+function print_version() {
```

Comment:
Lets run this somewhere around line 314 as well :) We have figured out we need this when reading buildlogs across rebuilders.